packages/notification-service/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@celo/contractkit": "0.1.5",
-    "@celo/utils": "^0.1.0",
+    "@celo/utils": "0.1.5-dev",
```

Comment:
Small gotcha here. The services that get deployed to App Engine can only depend on published versions of @celo packages, otherwise we're blocked from deploying. Can you use the latest published version here (or the one you're going to deploy)?